standby/Cargo.toml

```diff
@@ -13,6 +13,7 @@ repository = "https://github.com/twilight-rs/twilight.git"
 version = "0.1.0"
 
 [dependencies]
+dashmap = { default-features = false, version = "3" }
```

Comment:
isn't it better to use 4 because that doesn't have deadlock issues when used async?

Reply:
I think using release candidates is fine, we're not even in our own release candidate yet. From what I understand that Arcimon has said, v4 should be completely fine to use.

Reply:
DashMap 4 RC doesn't support `insert_if_not_exists` to imitate 3's Entry API that Standby uses, so this might be done later.